### PR TITLE
Update consul-template dependency to latest

### PR DESCRIPTION
**Pull request by notnoop**
To pick up the fix in
https://github.com/hashicorp/consul-template/pull/1231 .

### CODE REVIEW

File: client/allocrunner/taskrunner/template/template.go [hunk 1]
```diff
@@ -503,7 +503,7 @@ func templateRunner(config *TaskTemplateManagerConfig) (
 		return nil, nil, err
 	}
 
-	runner, err := manager.NewRunner(runnerConfig, false, false)
+	runner, err := manager.NewRunner(runnerConfig, false)
```
Comment (cgbaker) [inline client/allocrunner/taskrunner/template/template.go]: why the change to `once = true` here?

Comment (notnoop) [reply client/allocrunner/taskrunner/template/template.go]: I didn't notice the change in setting.  Looking at the code and the relevant PR changing the signature (https://github.com/hashicorp/consul-template/pull/1227), I still see `once` default to false.  When running a job, I see that `once` is set to false:

```
    2019/07/18 17:46:38.668349 [INFO] (runner) creating new runner (dry: false, once: false)
    2019/07/18 17:46:38.668615 [DEBUG] (runner) final config: {"Consul":{"Address":"","Auth":{"Enabled":false,"Username":"","Password":""},"Retry":{"Attempts":12,"Backoff":250000000,"MaxBackoff":60000000000,"Enabled":true},"SSL":{"CaCert":"","CaPath":"","Cert":"","Enabled":false,"Key":"","ServerName":"","Verify":true},"Token":"","Transport":{"DialKeepAlive":30000000000,"DialTimeout":30000000000,"DisableKeepAlives":false,"IdleConnTimeout":90000000000,"MaxIdleConns":100,"MaxIdleConnsPerHost":13,"TLSHandshakeTimeout":10000000000}},"Dedup":{"Enabled":false,"MaxStale":2000000000,"Prefix":"consul-template/dedup/","TTL":15000000000},"Exec":{"Command":"","Enabled":false,"Env":{"Blacklist":[],"Custom":[],"Pristine":false,"Whitelist":[]},"KillSignal":2,"KillTimeout":30000000000,"ReloadSignal":null,"Splay":0,"Timeout":0},"KillSignal":2,"LogLevel":"WARN","MaxStale":2000000000,"PidFile":"","ReloadSignal":1,"Syslog":{"Enabled":false,"Facility":"LOCAL0"},"Templates":[{"Backup":false,"Command":"","CommandTimeout":30000000000,"Contents":"hello\n  ","CreateDestDirs":true,"Destination":"/private/var/folders/r6/346cfqyn76b_lx1nrcl5278c0000gp/T/NomadClient459208508/e7176927-a54e-659b-01c8-aab4d1fc41f9/redis/local/file.yml","ErrMissingKey":false,"Exec":{"Command":"","Enabled":false,"Env":{"Blacklist":[],"Custom":[],"Pristine":false,"Whitelist":[]},"KillSignal":2,"KillTimeout":30000000000,"ReloadSignal":null,"Splay":0,"Timeout":30000000000},"Perms":420,"Source":"","Wait":{"Enabled":false,"Min":0,"Max":0},"LeftDelim":"{{","RightDelim":"}}"}],"Vault":{"Address":"","Enabled":false,"Grace":15000000000,"Namespace":"","RenewToken":false,"Retry":{"Attempts":12,"Backoff":250000000,"MaxBackoff":60000000000,"Enabled":true},"SSL":{"CaCert":"","CaPath":"","Cert":"","Enabled":true,"Key":"","ServerName":"","Verify":true},"Transport":{"DialKeepAlive":30000000000,"DialTimeout":30000000000,"DisableKeepAlives":false,"IdleConnTimeout":90000000000,"MaxIdleConns":100,"MaxIdleConnsPerHost":13,"TLSHandshakeTimeout":10000000000},"UnwrapToken":false},"Wait":{"Enabled":false,"Min":0,"Max":0},"Once":false}
```

Where do you see the value being changed?

Comment (cgbaker) [reply client/allocrunner/taskrunner/template/template.go]: Apologies, I think I got my wires crossed looking at different branches.
